src/Service/NotificationService.php

```diff
@@ -104,7 +113,19 @@ public function notificationHandler(array $data, array $configuration): array
             return ['response' => new Response($response, $exception->getCode(), ['Content-type' => 'application/json'])];
         }
 
-        $this->syncService->synchronize($synchronization);
+        try {
+            $response = $this->callService->call(
+                $synchronization->getSource(),
+                $synchronization->getEndpoint(),
+                'GET'
+            );
+        } catch (\Exception $exception) {
+            // Todo set error log
```

Comment:
Maybe add a loggerInterface log here, so we have error logs if we ever want to debug this online at some point